analyses/OH_cd_2010/doc_OH_cd_2010.md

```diff
@@ -1,15 +1,15 @@
 # 2010 Ohio Congressional Districts
 
 ## Redistricting requirements
-In Ohio, districts must:
+In Ohio, districts must, Under HB 319 https://legiscan.com/OH/text/HB319/2011:
```

Comment:
let's write this as 
`under [HB 319](https://legiscan.com/OH/text/HB319/2011):`

That will make "HB 319" a link for people to click.